### DISCUSSION TITLE
Ingress/Egress Gateway

### DISCUSSION THREAD

**Discussion by z24500112** [gravitl/netmaker#1649] [Q&A]
![image](https://user-images.githubusercontent.com/27390674/195294771-ad5bfd91-1102-48d8-abed-b72b23799377.png)
I want to ask, why I can't go from Ingress to Egress, currently using ext. Client wants to go from Ingress to NAT Egress and go out.

I'm sorry, I'm not familiar with the Internet, please help me
The following is the Network Graph
![image](https://user-images.githubusercontent.com/27390674/195296176-2a64989f-09c1-4c2a-aafe-0b200f8ba652.png)

**Comment by maulik9898** [comment]
I am also facing similar issue. I have two nodes. one is ingress and other one is egress. I want my internet traffic to go like this.
ext client -> node 1 (ingress) -> node 2 (egress) -> internet. setting 0.0.0.0/0 on egress does not work. my traffic still get routed thru node -1 to internet
